ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1433,20 +1464,74 @@ The configured NIC will be used to access the compute node.\n\n"
 		fi
 	done
 
-	promptvlan "(admin) VLAN ID" "$admin_vlan_id" "admin_vlan_id"
-	admin_vlan_id="$val"
-
-	printheader "Networking - Continued"
-	message=""
+	if [[ $ADVANCED_INSTALL -eq 1 ]]; then
+		promptvlan "(admin) VLAN ID" "$admin_vlan_id" "admin_vlan_id"
+		admin_vlan_id="$val"
+	else
+		admin_vlan_id=""
+	fi
 
+	printheader "Networking - Network tags"
+	message="
+By using additional network tags, you can easily identify multiple physical
+network interconnections between compute nodes.
+There are 3 configurable network tags: external, internal and storage.
+One physical NIC can have multiple network tags.\n\n"
 	if [[ $(getanswer "skip_instructions") != "true" ]]; then
 		printf "$message"
 	fi
 
+	# don't repeat selecting nic tags if they already have been defined in previous run
+	if [[ -z "${!nictag_macs[@]}" ]]; then
+		promptbool "\nAdd another network tag? (y/n)" "y"
+		printf "\n"
+		if [[ "${val}" -eq 1 ]]; then
+			for ntag in external internal storage; do
+				promptbool "Add tag \"${ntag}\"? (y/n)" "n"
```

Comment:
Refactored. It should work with an answer file. But I'm afraid there are other parts of the prompt-config that will currently not work with it. E.g: [this](https://github.com/erigones/esdc-factory/blob/167a68bfcf79610240b188165aa26930e6c9b4e3/ansible/templates/usb/scripts/prompt-config.sh.j2#L1410) or [this](https://github.com/erigones/esdc-factory/blob/167a68bfcf79610240b188165aa26930e6c9b4e3/ansible/templates/usb/scripts/prompt-config.sh.j2#L1441).